src/azure-cli/azure/cli/command_modules/privatedns/commands.py

```diff
@@ -56,22 +56,22 @@ def load_command_table(self, _):
     for record in supported_records:
         with self.command_group('network private-dns record-set {}'.format(record), network_privatedns_record_set_sdk) as g:
             g.show_command('show', 'get', transform=transform_privatedns_record_set_output, table_transformer=transform_privatedns_record_set_table_output)
-            g.command('delete', 'begin_delete', confirmation=True)
+            g.command('delete', 'delete', confirmation=True)
```

Comment:
Should not add `begin_` prefix as `delete` and `create_or_update` commands for `record-set` are not long running operations. You can find the method name in SDK source code.